client/src/pages/SignUp.vue

```diff
@@ -0,0 +1,185 @@
+        <v-form ref="form">
+          <v-row class="d-flex justify-center" no-gutters>
+            <v-col class="d-flex justify-center" style="max-width: 100px; width: 100px;">
+              <v-img
+                contain
+                src="@/assets/logo.png"
+                style="width: 100%; height: auto;"
+              />
+            </v-col>
+          </v-row>
+
+          <v-row class="d-flex justify-center" no-gutters>
+            <v-col class="d-flex justify-center" style="max-width: 80px; width: 80px;">
+              <p class="mt-2 text-h5 text-grey-darken-2" variant="h5">Sign up</p>
+            </v-col>
+          </v-row>
+
+          <v-row>
+            <v-col>
+              <v-text-field
+                v-model="newUser.first_name"
+                density="compact"
+                placeholder="First Name"
+                prepend-inner-icon="mdi-account-outline"
+                :rules="nameRules"
+                variant="outlined"
+              />
+              <v-text-field
+                v-model="newUser.last_name"
+                density="compact"
+                placeholder="Last Name"
+                prepend-inner-icon="mdi-account-outline"
+                :rules="nameRules"
+                variant="outlined"
+              />
+              <v-text-field
+                v-model="newUser.email"
+                density="compact"
+                placeholder="Email"
+                prepend-inner-icon="mdi-email-outline"
+                :rules="emailRules"
+                variant="outlined"
+              />
+              <v-text-field
+                v-model="newUser.password"
+                :append-inner-icon="visible ? 'mdi-eye-off' : 'mdi-eye'"
+                density="compact"
+                placeholder="Password"
+                prepend-inner-icon="mdi-lock-outline"
+                :rules="passwordRules"
+                :type="visible ? 'password' : 'text'"
+                variant="outlined"
+                @click:append-inner="visible = !visible"
+              />
+            </v-col>
+          </v-row>
+
+          <v-row>
+            <v-col class="d-flex justify-center">
+              <v-btn
+                block
+                class="mb-8 w-100"
+                color="primary"
+                :disabled="!isFormValid"
+                :loading="loading"
+                size="large"
+                @click="RegisterNewUser"
+              >
+                Register
+              </v-btn>
+            </v-col>
+          </v-row>
+
+          <v-divider />
+          <br>
+
+          <!-- <v-row>
+                <v-col cols="6" class="d-flex justify-center">
+                    <LoginGithub/>
+                </v-col>
+                <v-col cols="6" class="d-flex justify-center">
+                    <LoginTFT/>
+                </v-col>
+                </v-row> -->
+
+          <br>
+          <v-divider />
+          <br>
+
+          <AlreadyHaveAnAccount />
```

Comment:
We'll use the new name
```suggestion
          <Login />
```